WcaOnRails/app/models/competition.rb

```diff
@@ -507,7 +507,7 @@ def warnings_for(user)
         warnings = championship_warnings.merge(warnings)
       end
 
-      if has_fees? && !connected_stripe_account_id
+      if has_fees? && !CompetitionPaymentIntegration.stripe_connected?(self) && !CompetitionPaymentIntegration.paypal_connected?(self)
```

Comment:
Why not `!competiiton_payment_integrations.exists?`

Reply:
Much better, thanks!